Spanner/tests/Unit/BackupTest.php

```diff
@@ -105,24 +107,27 @@ public function testCreate()
             Argument::withEntry('backupId', self::BACKUP),
             Argument::withEntry('backup', [
                 'database' => DatabaseAdminClient::databaseName(self::PROJECT_ID, self::INSTANCE, self::DATABASE),
-                'expireTime' => $this->expireTime->format('Y-m-d\TH:i:s.u\Z')
-            ])
+                'expireTime' => $this->expireTime->format('Y-m-d\TH:i:s.u\Z'),
+            ]),
+            Argument::withEntry('versionTime', $this->versionTime->format('Y-m-d\TH:i:s.u\Z'))
         ))
             ->shouldBeCalled()
             ->willReturn([
                 'name' => 'my-operation'
             ]);
 
         $this->backup->___setProperty('connection', $this->connection->reveal());
-        $op = $this->backup->create(self::DATABASE, $this->expireTime);
+        $op = $this->backup->create(self::DATABASE, $this->expireTime, [
```

Comment:
Could we also test that the restored database `RestoreInfo.BackupInfo.versionTime` is the one used in the backup?